Playbooks/playbook-Frontline.Cloud_-_90_Day_Old_Critical_Vulnerabilities_Found.yml

```diff
@@ -0,0 +1,179 @@
+id: 6f8e5d0b-edbd-4d40-8d2d-dfa6f084ac0f
```

Comment:
please update the ID to the name of the playbook

Reply:
Changes made